### ISSUE TITLE
how can i flatten a tensor and concat three tensors

### ISSUE THREAD

**Issue by lxiao217** [NVIDIA/TensorRT#1944]
## Environment

**TensorRT Version*8.0.1*: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)*3.8*: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

13*80*80    13*40*40  13*20*20
(flatten)
13*6400     13*1600      13*400
(concat)
13*8400
(transpose)
8400*13

you can ignore the first dimension 1;
i want to flatten the tensor 13*80*80 to 13*6400, can then concat with the other two tensors, how can i implement it,
thanks for your reply.

**Comment by lxiao217** [comment]
i use c++ not python, but i cann't modify my content.
13x80x80 13x40x40 13x20x20
(flatten)
13x6400 13x1600 13x400
(concat)
13x8400
(transpose)
8400x13
 the multiply symbol star was removed by html system.

**Comment by zerollzeng** [comment]
https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#shuffle-layer and https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#concatenation-layer should be helpful to you

**Comment by nvpohanh** [comment]
Closing due to >14 days without activity. Please feel free to reopen if the issue still exists. Thanks
